.github/workflows/build-container-image.yml

```diff
@@ -0,0 +1,61 @@
+name: publish-container-image
```

Comment:
as a final step, rename this file to match the job name `publish-container-image`;)